docs/troubleshooting.rst

```diff
@@ -24,8 +24,37 @@ retaining the database::
   git submodule sync --recursive && git submodule update --init --recursive
   docker-compose pull
   docker-compose build --pull
-  docker-compose up
+  docker-compose up -d
```

Comment:
I go back and forth. I find the full logs to be distracting, so instead I use ``make up`` which runs ``docker-compose up -d``, and then run ``docker-compose logs -f worker`` (or a similar command) when I need to watch the logs from a particular container.

This seems to be a 1 to 1 vote, so I'll go back to the previous version which was reviewed and accepted.